.github/workflows/indexer-build-and-push-testnet.yml

```diff
@@ -4,8 +4,8 @@ on: # yamllint disable-line rule:truthy
   push:
     branches:
       - releases/testnet
-      - 'release/[a-z]+/v[0-9]+.[0-9]+.x'  # e.g. release/ibctestnet/v1.0.x
-      - 'release/v[0-9]+.[0-9]+.x'  # e.g. release/v0.0.x
+      - 'release/[a-z]+/0.[0-9]+.x'  # e.g. release/indexer/v0.1.x
+      - 'release/[a-z]+/v[0-9]+.x'  # e.g. release/indexer/v1.x
```

Comment:
To confirm, we explicitly still want build and push workflow on every push to `main` right?

Reply:
Yeah. I don't know if we really use this environment anymore tbh. I'll just put main for now.